client/assets/stylesheets/shared/_loading.scss

```diff
@@ -16,6 +16,24 @@
 	}
 }
 
+.loading {
```

Comment:
I haven't really tested, but this `loading` class seems to be in use in various existing places already -- are we deliberately applying this new styling to all of them?

Reply:
I found that I needed to add it here to be able to load styles during calypso loading. Indeed in the same file you can find the other loading components.

I just realized how general the class name is, so it will be used elsewhere. Let me update it to something else.

Reply:
Done!